e2e/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../tsconfig.base.json",
+  "compilerOptions": {
+    "types": ["cypress", "@testing-library/cypress"],
+    "paths": {
+      "e2e/support/*": ["./support/*"],
+      "*": ["../frontend/src/*"]
```

Comment:
Are you 100% sure we will never need anything from the `.frontend/test/`?

Reply:
I'm not 100% sure we **will**. But this could be adjusted when needed. By the way, it doesn't sound that enticing to me if we need to import anything from `frontend/test` in E2E tests, since they're quite different.